src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -73,6 +75,15 @@
     public static final String IDP_METADATA_FILE = "idp.metadata_file";
     public static final String IDP_METADATA_CONTENT = "idp.metadata_content";
 
+    private static final String OPENDISTRO_PREFIX = "/_opendistro/_security/";
```

Comment:
Minor nit: lets call this `LEGACY_OPENDISTRO_PREFIX` for all opendistro prefixes. 
```suggestion
    private static final String LEGACY_OPENDISTRO_PREFIX = "/_opendistro/_security/";
```
Ref: https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#rest-apis

Reply:
Fixed